### ISSUE TITLE
Soccer as Alternative Label for Football

### ISSUE THREAD

**Issue by SportsOpenData** [openactive-archive/activity-list-old#9]
Hi Guys,
i've seen the activity list feed in json.

Just a question about "Soccer": is it valid to consider it as alternative of "Football" activity?

Really thanks a lot.
Rgds.
Paolo.

**Comment by ldodds** [comment]
Yes, that's valid. The synonyms and description fields aren't really complete yet, so if you have suggestions then please file them here.
